### DISCUSSION TITLE
fprettify support for -c option

### DISCUSSION THREAD

**Discussion by mjreno** [krvajal/vscode-fortran-support#572] [Q&A - deprecated see Fortran Discourse] [answered]
Hey All!

I'm trying to set up fprettify formatting within Modern Fortran and am having trouble using the -c  (config file) option.  I'm not sure if this should be supported but the error I'm seeing suggests that perhaps it is. 

The relevant settings I am using are:
```json
"fortran.formatting.formatter": "fprettify",
"fortran.formatting.fprettifyArgs": ["-c C:\\Users\\mreno\\fprettify.yaml"],
```

The formatter works fine if I use options other than "-c".  However, when using the above settings a pop up opens with the error "Unable to open config file: C:\Users\mreno\prettify.yaml. Error: Invalid argument."  If I change to a Unix style path for the argument there is a different error- file not found.  It is a valid yaml file path which does work with fprettify on the command line.

Thank you for any thoughts on this and happy to provide additional information.

Mike

**Comment by gnikit** [comment, marked as answer]
Hi @mjreno, you need to split the argument flag and the actual argument e.g.

```json
"fortran.formatting.fprettifyArgs": ["-c", "C:\\Users\\mreno\\fprettify.yaml"],
```

**Reply by mjreno** [reply]
Thanks for the quick response @gnikit.
